src/_codux/boards/image-test/image-test.board.tsx

```diff
@@ -0,0 +1,8 @@
+import { createBoard } from '@wixc3/react-board';
```

Comment:
what is that?

Reply:
It was a board to check something, will delete